Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -198,6 +204,8 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
                 try container.encode(boolean)
             case .option(let option):
                 try container.encode(option)
+            case .geoJSONObject(let object):
```

Comment:
What if we just named it `geoJSON`? The word "object" always makes me think I'm dealing with an instance of a class.

Reply:
this is feedback for Turf too